File: src/main/java/com/github/jactor/web/controller/AboutController.java
package com.github.jactor.web.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.servlet.ModelAndView;

@Controller
public class AboutController {

    @GetMapping(value = "/about")
    public ModelAndView get() {
        return new ModelAndView("about");
    }
}

File: web/src/main/kotlin/com/github/jactor/web/controller/AboutController.kt
package com.github.jactor.web.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.servlet.ModelAndView

@Controller
class AboutController {
    @GetMapping(value = ["/about"])
    fun get(): ModelAndView {
        return ModelAndView("about")
    }
}
